app/controllers/application_controller.rb

```diff
@@ -21,8 +22,27 @@ def user_with_groups
   end
   helper_method :user_with_groups
 
+  def add_root_breadcrumb
+    @breadcrumbs = []
```

Comment:
I don't understand why we need to have these in the controller?  Can they just be part of the view component?

Reply:
I'm not quite sure how we'd implement this functionality easily in the breadcrumb view component, since each controller action is adding different items to the breadcrumb nav before we get to the rendering (and the welcome controller is turning off breadcrumb nav completely)?

Reply:
Refactored to remove from controllers as discussed